src/libraries/Microsoft.PowerFx.Core/Utils/TokenUtils.cs

```diff
@@ -12,6 +13,7 @@ namespace Microsoft.PowerFx.Core.Utils
 {
     internal static class TokenUtils
     {
+        [Obsolete]
         internal static FormulaType GetFormulaType(this IdentToken token)
```

Comment:
>GetFormulaType [](http://example.com/codeflow?start=36&length=14)

Since this is internal, can we just remove it? I don't think Power Apps uses this.